src/components/side_panel/pivot/pivot_side_panel/pivot_side_panel_store.ts

```diff
@@ -28,15 +28,6 @@ export class PivotSidePanelStore extends SpreadsheetStore {
     super(get);
   }
 
-  handle(cmd: Command) {
-    switch (cmd.type) {
-      case "UPDATE_PIVOT":
-        if (cmd.pivotId === this.pivotId) {
-          this.getters.getPivot(this.pivotId).init();
-        }
-    }
-  }
```

Comment:
Was that always useless ? I thought it would be useful if the pivot had no pivot formula in any sheet, but it seems buggy as hell .... If you rename the fields in the gris, the side panel won't update and mark them as invalid if you do'nt have a pivot formula somewhere. Not really related to your task tho

Reply:
But I'll revert the commit just for the sake of being extra safe in stable